features/old/api/features.feature

```diff
@@ -8,59 +8,80 @@ Feature: Features :)
     And an application plan "Basic" of provider "foo.3scale.localhost"
     Given current domain is the admin domain of provider "foo.3scale.localhost"
 
-  @javascript @wip
+  @javascript
   Scenario: Create a feature
     When I log in as provider "foo.3scale.localhost"
     And I go to the edit page for plan "Basic"
     And I follow "New feature"
     And I fill in "Name" with "Free T-shirt"
+    And I fill in "System name" with "tee"
     And I fill in "Description" with "T-shirt with logo of our company for free."
-    And I press "Create Feature"
-    Then I should see enabled feature "Free T-shirt"
-    And provider "foo.3scale.localhost" should have feature "Free T-shirt"
-    And feature "Free T-shirt" should be enabled for plan "Basic"
+    And I press "Save"
+    Then I should see "T-shirt with logo of our company for free."
 
-  # TODO: Test that the "No features yet" notice dissapears when first feature is created,
-  #       and appears when last one is deleted.
+  @javascript
+  Scenario: "No features yet" notice disappears when first feature is created
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
+    Given the provider has "service_plans" switch allowed
+    And a feature "50% less bugs" of provider "foo.3scale.localhost"
+    When I log in as provider "foo.3scale.localhost"
+    And I go to the service plans admin page
+    And I follow "Basic"
+    Then I should not see "This plan has no features yet."
+
+  @javascript
+  Scenario: "No features yet" notice appears when last one is deleted or no feature present.
```

Comment:
"when last one is deleted" flow is not covered, which is OK, but then the scenario title shouldn't say so :)